src/pudl/metadata/resources/ferc1_eia_record_linkage.py

```diff
@@ -88,7 +91,9 @@
         "field_namespace": "eia",
     },
     "out_eia__yearly_generators_by_ownership": {
-        "description": "A mega table of all EIA generators with ownership integrated.",
+        "description": {
+            "additional_summary_text": "all EIA generators with ownership integrated.",
+        },
```

Comment:
oof this is oos for this pr 100% but this is too short for what this table is and has no pk.

Reply:
also also
```suggestion
        "description": {
            "additional_summary_text": "all EIA generators with ownership integrated.",
            "usage_warnings": ["aggregation_hazard"],
        },
```